client/ayon_zbrush/api/pipeline.py

```diff
@@ -119,16 +118,9 @@ def save_workfile(self, filepath=None):
         return filepath
 
     def get_current_workfile(self):
-        work_dir = get_workdir()
-        txt_dir = os.path.join(
-            work_dir, ".zbrush_metadata").replace(
-                "\\", "/"
-        )
-        with open (f"{txt_dir}/current_file.txt", "r") as current_file:
-            content = str(current_file.read())
-            filepath = content.rstrip('\x00')
-            current_file.close()
-            return filepath
+        if not os.getenv("CURRENT_ZPR", ""):
+            return ""
+        return os.environ["CURRENT_ZPR"]
```

Comment:
This is exactly the same as:
```suggestion
        return os.getenv("CURRENT_ZPR", "")
```